cmd/root.go

```diff
@@ -125,7 +125,7 @@ func validateServiceWithLabel(service string) error {
 		}
 	} else {
 		if !validServicePathFormatWithLabel.MatchString(service) {
-			return fmt.Errorf("Failed to validate service name '%s'.  Only alphanumeric, dashes, forwardslashes, fullstops and underscores are allowed for service names, and colon followed by a label name", service)
+			return fmt.Errorf("Failed to validate service name '%s'.  Only alphanumeric, dashes, forwardslashes, fullstops and underscores are allowed for service names, and colon followed by a label name. Service names must not start or end with a forwardslash or colon", service)
```

Comment:
Why does this one say colon but not the above one? Because the label can have colons?

Reply:
Yep! Exactly

Difference in the regex's:
```
validServicePathFormat          = regexp.MustCompile(`^[\w\-\.]+(\/[\w\-\.]+)*$`)
validServicePathFormatWithLabel = regexp.MustCompile(`^[\w\-\.]+((\/[\w\-\.]+)+(\:[\w\-\.]+)*)?$`)
```
and the [tests show that as well](https://github.com/segmentio/chamber/blob/ade72496d6ae1391006a218a569862dd7ed07f20/cmd/root_test.go#L112-L121)

Reply:
Same nitpick for typos as above

```suggestion
			return fmt.Errorf("Failed to validate service name '%s'. Only alphanumeric, dashes, forward slashes, full stops and underscores are allowed for service names, and colon followed by a label name. Service names must not start or end with a forward slash or colon", service)
```

Reply:
Thanks for pointing it out! Added in https://github.com/segmentio/chamber/pull/347/commits/843d594527ac20891d5f088273c9c1630edbb38a (found some other usages of `fullstops` and double spaces)